c7n/resources/account.py

```diff
@@ -1383,114 +1382,3 @@ def process(self, resources):
             client.put_public_access_block(
                 AccountId=r['account_id'],
                 PublicAccessBlockConfiguration=config)
-
-
-@filters.register('glue-security-config')
-class GlueEncryptionEnabled(MultiAttrFilter):
```

Comment:
this one has to remain on the account, as it was in a previous release, we'll need to duplicate it

Reply:
Got it,  addressed!

Reply:
You mean duplicate the functionality and not copy the filter to resource: glue-catalog, right? I functionality can be achieved using value filter